### PR TITLE
Fix typos and trim whitespace

### PR DESCRIPTION
**Pull request by fdmarcin**
Just some quick fixes for things I noticed.

The game is very well written and I played it after reading _Skyrim rendered in text_. Keep up the work!

### CODE REVIEW

**Comment by filiph** [comment]
Thanks and sorry for letting your pull request stand this long. 

Those are some really important fixes — I hate that I made so many stupid typos. I'm putting merging this off, though, since I'm in the middle of a rewrite, focusing on the backend stuff (read: not the writing). This wouldn't normally be a problem, but I have the project set up so that the "source of truth" for the texts is in a Google Docs folder (for easier collaboration) — which is great but it means that all your suggestions will need to be tracked down in Google Docs and fixed there as well.

TL;DR: I'll get back to this. I really appreciate the contribution!

**Comment by fdmarcin** [comment]
Hey, thanks for the response. Glad to be of help!

**Comment by filiph** [comment]
I finally got to this, and deployed. Thanks again. And thanks for @xrd for nudging me with his own PR.
